src/test/java/trello/steps/CardSteps.java

```diff
@@ -71,4 +75,41 @@ public void verifyTheNameCard() {
         String actualTitle = boardPage.getCardTitle(context.getList().getTitle(), context.getCard().getTitle());
         Assert.assertEquals(context.getCard().getTitle(), actualTitle);
     }
+
+    /**
+     * Select a card of board.
```

Comment:
Please, change the verb in third person.

Reply:
Solved